tests/autoyast/prepare_cloned_profile.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
```

Comment:
Same here, let's use actual year =)

Reply:
oh! I didn't update myself internally with that, I thought it was ok after checking it, thanks.